packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -146,16 +149,19 @@ class _CupertinoButtonState extends State<CupertinoButton> with SingleTickerProv
     _setTween();
   }
 
-  void _handleTapDown(PointerDownEvent event) {
-    _animationController.animateTo(1.0, duration: kFadeOutDuration);
+  Future<Null> _handleTapDown(PointerDownEvent event) async {
+    await _lastAnimation;
+    _lastAnimation = _animationController.animateTo(1.0, duration: kFadeOutDuration);
```

Comment:
That only happens when the tap might be disambiguated as something else though, right?